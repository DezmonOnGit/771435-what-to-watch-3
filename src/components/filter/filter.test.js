import React from "react";
import renderer from "react-test-renderer";
import Filter from "./filter.jsx";

const filter = [`test-filter-1`, `test-filter-2`, `test-filter-3`];
const classes = {
  list: `test-list`,
  item: `test-item`,
  itemActive: `test-item--active`,
  link: `test-link`
};

describe(`Render filter`, () => {
  it(`should render <Filter />`, () =>{
    const tree = renderer.create(
        <Filter
          filter={filter}
          classes={classes}
        />
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
