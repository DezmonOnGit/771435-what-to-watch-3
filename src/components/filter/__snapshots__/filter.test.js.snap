// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render filter should render <Filter /> 1`] = `
<ul
  className="test-list"
>
  <li
    className="test-item test-item--active"
  >
    <a
      className="test-link"
      href="#"
    >
      test-filter-1
    </a>
  </li>
  <li
    className="test-item"
  >
    <a
      className="test-link"
      href="#"
    >
      test-filter-2
    </a>
  </li>
  <li
    className="test-item"
  >
    <a
      className="test-link"
      href="#"
    >
      test-filter-3
    </a>
  </li>
</ul>
`;
